.github/workflows/stalebot.yaml

```diff
@@ -0,0 +1,40 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          stale-issue-message: 'This issue has been automatically marked as stale due to lack of activity. You can remove the stale label or comment. Otherwise, this issue will be closed in 7 days. Thank you!'
+          close-issue-message: 'We are closing this issue due to lack of activity. Feel free to reopen it if you can provide more information. Thank you!'
+       
+          # Don't process PRs 
+          days-before-stale: -1
+
+          # Process only issues
+          days-before-issue-stale: 14
+          days-before-issue-close: 7
```

Comment:
I think we should be less aggressive with those numbers... perhaps 30-30 or 30-15